0.10/contrib/etc/generate_container_user

```diff
@@ -0,0 +1,19 @@
+# Set current user in nss_wrapper
+USER_ID=$(id -u)
+GROUP_ID=$(id -g)
+
+if [ x"$USER_ID" != x"0" -a x"$USER_ID" != x"1001" ]; then
+
+    NSS_WRAPPER_PASSWD=/opt/app-root/etc/passwd
```

Comment:
we don't want to export that until we successfully create the new passwd file.  i know, unlikely to be an issue, but it seems best not to.